templates/support/esm.html

```diff
@@ -0,0 +1,109 @@
+{% extends "support/base_support-v1.html" %}
+
+{% block title %}Ubuntu 12.04 LTS Extended Security Maintenance{% endblock %}
+
+{% block second_level_nav_items %}
+  {% include "templates/_nav_breadcrumb-v1.html" with section_title="Support" page_title="ESM" %}
+{% endblock second_level_nav_items %}
+
+{% block content %}
+
+<div class="p-strip is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>Ubuntu 12.04 Extended Security Maintenance</h1>
+      <p>Following the end-of-life of Ubuntu 12.04 LTS in April 2017, Canonical is offering Ubuntu 12.04 ESM (Extended Security Maintenance). ESM provides ongoing security fixes for the kernel and essential user space packages through April 2019 via a secure, private archive available exclusively to Ubuntu Advantage customers.</p>
+      <p><a href="https://buy.ubuntu.com/" class="p-button--brand">Buy Ubuntu Advantage</a></p>
+      <p><a href="/support">Learn more about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center">
+      <img src="{{ ASSET_SERVER_URL }}9e59756d-shield-ESM.svg" alt="" />
+    </div>
+  </div>
+</div>
+
+<div class="p-strip--grey is-bordered p-phone-contact">
```

Comment:
Can you push back on this new strip type? We can get pretty close using patterns: https://pastebin.canonical.com/188411/